Cargo.toml

```diff
@@ -61,6 +61,7 @@ vdso = { version = "0.2", optional = true, default-features = false }
 x86_64 = { version = "0.14.9", optional = true, default-features = false }
 
 # binary dependencies
+[target.'cfg(target_os = "linux")'.dependencies]
 # FIXME: The enarx-exec-wasmtime-bin crate should be merged with enarx-exec-wasmtime again, see https://github.com/enarx/enarx/issues/1798
 enarx-exec-wasmtime-bin = { version = "0.4.0", path = "crates/exec-wasmtime-bin", artifact = "bin", target = "x86_64-unknown-linux-musl", default-features = false }
```

Comment:
Is it possible to express the targets as linux but without the arch? Thinking about Rasp Pi.